bridge/client/app/_views/ktb-sequence-view/ktb-sequence-view.component.html

```diff
@@ -35,6 +35,7 @@
             [events]="getFilteredSequences(project.sequences)"
             [selectedEvent]="currentSequence"
             (selectedEventChange)="selectSequence($event)"
+            (loadOldSequencesClicked)="loadSequenceMetadata(project.projectName)"
```

Comment:
Actually, we don't need this. The component listens to `this.dataService.sequencesUpdated` and so does `ktb-sequence-view.component`. Additionally, I wouldn't even send another call to the metadata. Just retrieving that in an interval is fine. We just need to update the current metadata with the sequences we have.

Reply:
I keep it as it is